rules/linux/execution_shell_evasion_linux_binary.toml

```diff
@@ -0,0 +1,104 @@
+[metadata]
+creation_date = "2022/05/06"
+maturity = "production"
+updated_date = "2022/05/09"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary(s) abuse to breakout out of restricted shells or environments by spawning an interactive system
+shell. The linux utility(s) activity of spawning shell is not a standard use of the binary for a user or system
+administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their
+access.
```

Comment:
```suggestion
Identifies Linux binary(s) abuse to breakout out of restricted shells or environments by spawning an interactive system
shell. The linux utility(s) activity of spawning shell is not a standard use of the binary for a user or system
administrator. It may indicates an attempt to improve the capabilities or stability of an adversary access.
```